styles.css

```
@@ -1 +1,82 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #FEFAE0;
+}
+
+.container {
+    margin: 20px;
+    background-color: #FEFAE0;
```

@abouman76 -   de background-color wordt omschreven in hsl (zie onderstaande code)
 ```suggestion
    background-color: hsl(32, 100%, 93%);
```